package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-r9+UiPsezvyDSPb7IvRk4ypYs2s=",
       "dev": true,
       "requires": {
-        "acorn": "^3.0.4"
+        "acorn": "3.3.0"
```

Comment:
did you mean to change all the transitive dependency versions? 
i doubt you did..

revert this file?